dev-lang/rapydscript-ng/rapydscript-ng-0.7.22.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+if [[ ${PV} == 9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/kovidgoyal/rapydscript-ng"
+	# this is nodejs software, so possibly unfixable. I am sorry.
+	# Instead, we will have to generate this with the live ebuild and upload assets
+	# using EGIT_OVERRIDE_COMMIT for the tagged version we need.
+	RESTRICT="network-sandbox"
+	BDEPEND="app-arch/libarchive"
+
+	inherit git-r3
+else
+	SRC_URI="
+		https://github.com/kovidgoyal/rapydscript-ng/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+		https://help-me/rapydscript-ng-node_modules-${PV}.tar.xz
```

Comment:
I do need to figure out where precisely to host this. The version I generated from the live ebuild is
[rapydscript-ng-node_modules-0.7.22.tar.xz.txt](https://github.com/gentoo/gentoo/files/13046649/rapydscript-ng-node_modules-0.7.22.tar.xz.txt)

Reply:
I'll rehost it, thx